### ISSUE TITLE
Sound in call too loud

### ISSUE THREAD

**Issue by justfun23** [TheWhisp/android_device_samsung_wilcoxltexx#35]
I have a certain problem whenever I am in a call, the speaker is too high even if the volume it's down to the lowest possible value, if I bring it up to the maximum nothing happens it's like it's at max volume all the time. Please help, thank you.

**Comment by TheWhisp** [comment]
the earpiece or the speaker?

**Comment by justfun23** [comment]
The speaker. Also when using the headphones. It is annoying because people hear themselves while talking to me .

**Comment by TheWhisp** [comment]
So does this happens when you put the calls on the speaker ? The earpiece is the little speaker that is used in a normal [non speaker] call.

**Comment by justfun23** [comment]
No, when i'm not on speaker it acts like I am on speaker , it's just to loud and annoying, when I am put lowest volume it's loud. I have done an experiment, put it on the highest volume while being in a call without putting the speaking on and went to another room and still heard what the person was saying in the other room, the distance was significant , had the door shut. So you get the idea.

**Comment by TheWhisp** [comment]
well, yeah, it is a bit louder, however I wouldn't say THAT louder. The same happens on latest build?

**Comment by justfun23** [comment]
Yes.

**Comment by TheWhisp** [comment]
I just tested on the latest build and volume can be adjusted in both modes
